# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: netwerk/wifi/nsWifiMonitor.cpp
# Commit: 6cbe2549217c
# Full Hash: 6cbe2549217c7cc45f19dce0e064e8ad94601969
# Author: David Parks <daparks@mozilla.com>
# Date: 2023-07-21 04:09:26
# Description:
#   Bug 1842860: Increase stack size in WifiMonitor on MacOS r=haik,necko-reviewers,kershaw
#   
#   We have been seeing the stack issue from bug 1776210, but on MacOS 10.12 Sierra.  We are increasing stack size for all versions.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D183611
# ==============================================================================

diff -r b0f40ba976ad -r 6cbe2549217c netwerk/wifi/nsWifiMonitor.cpp
--- a/netwerk/wifi/nsWifiMonitor.cpp	Fri Jul 21 00:54:10 2023 +0300
+++ b/netwerk/wifi/nsWifiMonitor.cpp	Thu Jul 20 22:11:27 2023 +0000
@@ -248,14 +248,12 @@
 #else
     // If this ASSERT fails, we've increased our default stack size and
     // may no longer need to special-case the stack size on macOS.
-    static_assert(kMacOS13MonitorStackSize >
+    static_assert(kMacOSWifiMonitorStackSize >
                   nsIThreadManager::DEFAULT_STACK_SIZE);
 
     // Mac needs a stack size larger than the default for CoreWLAN.
     nsIThreadManager::ThreadCreationOptions options = {
-        .stackSize = nsCocoaFeatures::OnVenturaOrLater()
-                         ? kMacOS13MonitorStackSize
-                         : nsIThreadManager::DEFAULT_STACK_SIZE};
+        .stackSize = kMacOSWifiMonitorStackSize};
 #endif
 
     nsresult rv = NS_NewNamedThread("Wifi Monitor", getter_AddRefs(mThread),